<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEntry</class>
 <widget class="QWidget" name="AddEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>481</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Tilføj tidsplan</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Starttid </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sluttid</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuværende tidsplan</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hvilken dag ønskes indstilles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Mandag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tirsdag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Onsdag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Torsdag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fredag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lørdag</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Søndag</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Vælg start tidspunkt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="StartTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Vælg slut tidspunkt </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="EndTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Tilføj tidsplan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Annuller</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>comboBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>StartTime</zorder>
   <zorder>label_3</zorder>
   <zorder>EndTime</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>EndTime</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
